llvm/lib/AsmParser/LLLexer.cpp

```diff
@@ -987,6 +990,7 @@ lltok::Kind LLLexer::LexIdentifier() {
 lltok::Kind LLLexer::Lex0x() {
   CurPtr = TokStart + 2;
 
+  // TODO: Handle the DFP type here.
```

Comment:
```suggestion
```
Per the C23 draft, section 6.4.4.2, Floating constants, p2, hexadecimal floating constants are not supported for DFP types. I think this comment can be removed.

Reply:
Then I don't understand the text that comes in p5?